cin_validator/rules/cin2022_23/rule_8620.py

```diff
@@ -0,0 +1,111 @@
+from typing import Mapping
```

Comment:
if you could format this by running 'black /workspaces/CIN-validator/cin_validator/rules/cin2022_23' in the terminal, Gavin, that'd be really good so we don't get the linting error. I tend to use the filepath of the whole cin_validator module rather than just the rule file as it'll fix any other files with issues too.